artifactory/cli.go

```diff
@@ -1053,7 +979,7 @@ func GetCommands() []cli.Command {
 	})
 }
 
-func createArtifactoryDetailsByFlags(c *cli.Context, distribution bool) (*coreConfig.ServerDetails, error) {
+func CreateArtifactoryDetailsByFlags(c *cli.Context, distribution bool) (*coreConfig.ServerDetails, error) {
```

Comment:
Notice that this function includes ```if distribution```, although we're inside ```artifactory/cli.go```.
Here's what I suggest we do:
1. Make this function private again.
2. Create a similar function inside ```distribution/cli.go```, which includes only the distribution related code (without the if statement which is not needed anymore).
3. Also remove the if statement in the function in this file.